mealpy.py

```diff
@@ -1,126 +1,111 @@
 import getpass
 import json
-import requests
 import time
 
+import requests
 from apscheduler.schedulers.blocking import BlockingScheduler
 
 BASE_DOMAIN = 'secure.mealpal.com'
-BASE_URL = 'https://' + BASE_DOMAIN
-LOGIN_URL = BASE_URL + '/1/login'
-CITIES_URL = BASE_URL + '/1/functions/getCitiesWithNeighborhoods'
-MENU_URL = BASE_URL + '/api/v1/cities/%s/product_offerings/lunch/menu'
-RESERVATION_URL = BASE_URL + '/api/v2/reservations'
-KITCHEN_URL = BASE_URL + '/1/functions/checkKitchen3'
-
-LOGGED_IN_COOKIE = 'isLoggedIn'
```

Comment:
Same for this constant. Removing this introduces "magic string" in the code below.

Reply:
Moving it to a global doesn't unmagic the string, it's still a string literal that has no context. 

This isn't really a magic string though, it's part of the undocumented interface to mealpal cookie. If they had a real API, it would be documented as part of their openapi/REST interface as parameters to set before making request.

If you read line #30, it reads as "set key of 'isLoggedIn' to cookie", whereas before was "set key of LOGGED_IN_COOKIE to cookie, where LOGGED_IN_COOKIE has the value of 'isLoggedIn'".